README.md

```diff
@@ -1,6 +1,13 @@
-# @TxnLab/use-wallet
+<p align="center"><img  width=100%  src="https://ipfs.algonode.xyz/ipfs/bafkreiexg66kttj75i6uv6khowp2ybvue4nwhvuhh24dobpniszqkbuxoq" /></p>
```

Comment:
Let's hold off on the logo for now. I tend to be very particular about branding and have some ideas for the logo that are still being fleshed out ;-)

I do like the extended description you provide below.

Reply:
https://ipfs.algonode.xyz/ipfs/bafkreiexg66kttj75i6uv6khowp2ybvue4nwhvuhh24dobpniszqkbuxoq

here is the image for reference - removed it from readme and replaced with simple textual description